TSRM/TSRM.c

```diff
@@ -10,7 +10,7 @@
    +----------------------------------------------------------------------+
 */
 
-#include "TSRM.h"
+#include "zend.h"
```

Comment:
We could introduce more callbacks in TSRM, currently there are only a couple of callbacks at thread creation. Some callbacks could be introduced for the tsrm startup/shutdown and thread shutdown. I'll do the callbacks for now, for cleaner TSRM API.